lib/RedisDB/Cluster.pm

```diff
@@ -153,17 +153,32 @@ contain 'host' and 'port' elements. Constructor will try to connect to nodes
 from the list and from the first node to which it will be able to connect it
 will retrieve information about all cluster nodes and slots mappings.
 
+=over 4
+
+=item password
+
+Password, if redis server requires authentication.
+
+=item database
+
+DB number to use. Specified database will be selected immediately after
+connecting to the server. Database changes when you sending I<select> command
+to the server. You can get current database using I<selected_database> method.
+Default value is 0.
```

Comment:
according to specification (https://redis.io/topics/cluster-spec):
> Redis Cluster does not support multiple databases like the stand alone version of Redis. There is just database 0 and the SELECT command is not allowed.
So that's not going to work (or at least it shouldn't according to the spec)